xmlbuilder/xmlbuilder-tests.ts

```diff
@@ -14,31 +17,31 @@ xml('test17', { headless: true }).ins({ 'pi': 'mypi', 'pi2': 'mypi2', 'pi3': nul
 xml('test17', { headless: true }).ins(['pi', 'pi2']).end();
 
 xml('test18', { headless: true })
-    .ins('renderCache.subset', '"Verdana" 0 0 ISO-8859-1 4 268 67 "#(),-./')
-    .ins('pitarget', () => 'pivalue')
-    .end();
+  .ins('renderCache.subset', '"Verdana" 0 0 ISO-8859-1 4 268 67 "#(),-./')
+  .ins('pitarget', () => 'pivalue')
+  .end();
```

Comment:
these whitespace changes are unnecessary. please revert to keep a cleaner history